_tut/docs/3x/index.md

```diff
@@ -27,6 +27,9 @@ title: Monix Documentation
   - [Observers and Subscribers](./reactive/observers.html)
   - [Consumer](./reactive/consumer.html)
   - [Javascript Event Listeners](./reactive/javascript.html)
+- Pull-based streaming
```

Comment:
Maybe "Push-based streaming" should be mentioned near "Reactive" bit above, so it is evident that two are similar yet slightly different?

Reply:
Observable isn't exactly pure push-based, because it does back-pressure.